gfx/drivers_context/vita_ctx.c

```diff
@@ -21,7 +21,9 @@
 static void vita_swap_interval(void *data, int interval)
 {
    (void)data;
-   (void)interval;
+#ifdef VITA
```

Comment:
There's no need to ifdef for vita. Change name for vita_gl_ctx because it is gl dependant

Reply:
the fact of being gl dependant means nothing. orbis is gl dependant too as well as switch.

Reply:
Right but I'm doing a vita_gxm_ctx so it's not the same.